src/vcWebFile.cpp

```diff
@@ -156,8 +156,7 @@ udResult vcWebFile_RegisterFileHandlers()
 #elif UDPLATFORM_EMSCRIPTEN
 void vcWebFile_OpenBrowser(const char *pWebpageAddress)
 {
-  // TODO: Open new tab with link
-  udUnused(pWebpageAddress);
+  MAIN_THREAD_EM_ASM(window.open(UTF8ToString($0)), pWebpageAddress);
```

Comment:
I'm not sure this is in the version of Emscripten that we use - I have not seen this before, any idea what version this came in?

Reply:
2017 I think? https://github.com/emscripten-core/emscripten/pull/5544